client/src/lib/error-telemetry.js

```diff
@@ -0,0 +1,68 @@
+/**
+* The error telemetry module handles sending error logging to the backend.
+*
+* NOTE: we _might_ want in the future to have only one Telemetry class and
+* handle both error and "metrics" telemetry, but let's start with a smaller
+* scope at least for now.
+*/
+const Tail   = require('tail').Tail;
+const fs     = require('fs');
+const logger = require('winston');
+
+const TARGET = '/tmp/test';
+
+class ErrorTelemetry {
+  constructor(app, options) {
+    this.app = app;
+    this.options = options;
+  }
+
+  setup() {
+    logger.info('Setting up error logging...');
+    var loggingFile = this.fileToWatch();
```

Comment:
Generally I've been avoiding the use of `var` in favor of the block-scoped [let](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let).

FWIW

Reply:
In this case it's even a `const` as it's never modified after that.

Reply:
I've fixed this and enabled the `no-var` rule in ESLint, see latest commits.